examples/app-demo/restaurant-backend/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use anyhow::Result;
+
+use slight_http_handler_macro::register_handler;
+
+wit_bindgen_rust::import!("../../../wit/http.wit");
+wit_error_rs::impl_error!(http::Error);
+
+wit_bindgen_rust::import!("../../../wit/mq.wit");
+wit_error_rs::impl_error!(mq::Error);
+
+fn main() -> Result<()> {
+    let router = http::Router::new()?;
+    let router_with_route = router
+        .post("/make_order", "handle_make_order")?
+        .get("/get_next_order", "handle_get_next_order")?;
+    let _ = http::Server::serve("0.0.0.0:3000", &router_with_route)?;
+    Ok(())
+}
+
+#[register_handler]
+fn handle_make_order(req: Request) -> Result<Response, Error> {
+    let mq = crate::mq::Mq::open("slight-restaurant").expect("failed to open message queue");
+    mq.send(&req.body.as_ref().unwrap())
+        .expect("failed to make order");
+    Ok(Response {
+        headers: Some(req.headers),
+        body: None,
+        status: 200,
+    })
+}
+
+#[register_handler]
+fn handle_get_next_order(req: Request) -> Result<Response, Error> {
+    let mq = crate::mq::Mq::open("slight-restaurant").expect("failed to open message queue");
+    Ok(Response {
+        headers: Some(req.headers),
+        body: Some(mq.receive().expect("failed to get next order")),
+        status: 200,
+    })
+}
+
+#[register_handler]
```

Comment:
Does this handler have a purpose? Is it for some illustrative purpose?

Reply:
It is a requirement rn~ as per: https://github.com/deislabs/spiderlightning/blob/a2bd250eac68550d1df1d6b357d7d2aa1f89818e/examples/http-demo/src/main.rs#L75-L77